core/src/components/alert/alert.scss

```diff
@@ -52,6 +52,10 @@
   align-items: flex-start;
 }
 
+:host ion-backdrop {
+  z-index: 0;
+}
```

Comment:
This was added because I was running into an issue where inline alerts would show the backdrop on top of the overlay. I figure the backdrop's default `z-index` of 2 isn't needed since the overlay wrapper around it gets its own massive z-index anyway.